### PR TITLE
pythonPackages.srvlookup: init at 2.0.0

### PR DESCRIPTION
**Pull request by mmlb**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/srvlookup/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+buildPythonPackage rec {
+  pname = "srvlookup";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1zf1v04zd5phabyqh0nhplr5a8vxskzfrzdh4akljnz1yk2n2a0b";
+  };
+
+  buildInputs = [ dnspython ];
```
Comment (jonringer) [inline pkgs/development/python-modules/srvlookup/default.nix:15]: python packages have to be propagated
```suggestion
  propagatedBuildInputs = [ dnspython ];
```

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

File: pkgs/development/python-modules/srvlookup/default.nix (hunk 1 above)
Comment (mmlb) [reply pkgs/development/python-modules/srvlookup/default.nix]: @jonringer thanks for spotting this. Seems I missed the notification for this comment. Fixed. PTAL.

**Comment by mmlb** [comment]
@jonringer please take another look

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 90087 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>3 packages built:</summary><br>- python27Packages.srvlookup<br>- python37Packages.srvlookup<br>- python38Packages.srvlookup</details>

**Pull request merged**